requirements.pinned.txt

```diff
@@ -1,20 +1,20 @@
 alembic==1.7.7
 amqp==5.1.1
-appnope==0.1.3
 arrow==1.2.3
 async-timeout==4.0.2
 attrs==22.1.0
 Babel==2.10.3
 backcall==0.2.0
 base32-lib==1.0.2
 billiard==3.6.4.0
+black==21.12b0
```

Comment:
it should be coming only from tests, so not here?

Reply:
I did a `pip install -e .` and it was installed, however I removed it :+1: